avocado_vt/loader.py

```diff
@@ -107,7 +108,10 @@ def _get_parser(self):
 
     def get_extra_listing(self):
         if self.args.vt_list_guests:
-            guest_listing(self.args)
+            args = copy.copy(self.args)
+            args.vt_config = None
+            args.vt_guest_os = None
```

Comment:
In the future, I'd be really happy to see this converted to `avocado vt-list-guests`